cross/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN curl -fsSL "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.1.21.zip"
 
 # ------------------------------------------------------------------------------------------------
 # Go support
-RUN GO_VERSION=1.16.1 && \
+RUN GO_VERSION=1.17.5 && \
```

Comment:
Just curious, did you have a case where you were running into issues with Go version? I believe we have moved away from using cross container everywhere in Crossplane ecosystem

Reply:
No, I just wanted to upgrade starting from crossplane-runtime and found this line. Should we remove it altogether if it's not used in Upbound as well?

Reply:
@muvaf I think we need to keep the cross container around for the time-being 👍🏻